### PR TITLE
Merge feature/DLOB-3223-Add-fileDownloadUrl-in-DJSF-Spec

### PR DESCRIPTION
**Pull request by fabian-bickel**
Add fileDownloadUrl in DJSF spec.

### CODE REVIEW

**Comment by ylapp1** [comment]
Just a question:
Is it correct that we need to increase the DJSF schema-version in the Motion-Code so that OnBoard knows that the new `fileDownloadUrl` property exists in the schedule JSON data?
https://github.com/CN-Consult/DiLoc/blob/6fa7b04f9ca265330f4398aa1cb12f2d7d9b9c49/backend/lib/rail/subsystems/mpi/scheduleformats/djsf/json/schedule.php#L48-L58

If yes we should adjust the schema description to tell that it is version `2.4.3` or `2.5.0` now
https://github.com/CN-Consult/djsf/blob/60e482393895dd5dd05f76dc074d4a7f1118f286/djsf.json#L5

And a changelog description should be added to README: https://github.com/CN-Consult/djsf/blob/60e482393895dd5dd05f76dc074d4a7f1118f286/README.md?plain=1#L88-L89

File: djsf.json [hunk 1]
```diff
@@ -43,6 +43,11 @@
 					"format":"date-time",
 					"required":true,
 					"description": "The timestamp when the schedule is created in ISO-8601 format. For example: 2022-12-02T10:20:39+00:00"
+				},
+				"fileDownloadUrl": {
+					"type":"string",
+					"required":false,
+					"description": "The URL with path to the API endpoint to download schedule audio files"
```
Comment (Shyru) [inline djsf.json]: Should we document here, how the filename of the file to be downloaded is appended to the url?

**Review by Shyru — CHANGES_REQUESTED** [review]
I would suggest to enhance the documentation a bit more.

**Review by Shyru — APPROVED** [review]
Looks good!

**Pull request merged**